lib/backend/k8s/k8s.go

```diff
@@ -223,6 +223,7 @@ func (c *KubeClient) ensureClusterType() (bool, error) {
 		}
 		value = existingValue
 	}
+	log.Debugf("Setting ClusterType to '%s'", value)
```

Comment:
Would now be a good time for us to start making use of [logrus' `withField`](https://github.com/sirupsen/logrus#example)?

```
log.WithField("ClusterType", value).Debug("Setting ClusterType")
```